src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx

```diff
@@ -200,6 +204,44 @@ Create a new login and grant the necessary permissions:
     ```
 
   </Collapser>
+
+  <Collapser
+    id="domain-user"
+    title="For Windows Group Managed Service Account"
+  >
+
+
+    1. Create a new login and grant `CONNECT` and `VIEW` permissions:
```

Comment:
Are they creating a new login on the SQL side, or on the new relic side? Where are they adding this snippet to grant connect and view permissions? @phillipsj

Reply:
This is for creating the login on the SQL Server side, not in New Relic.